### PR TITLE
Home page and feed changes

### LINKED ISSUE #1581: Member DAOs First on Alchemy Home Page
Eric, Eylon, and I view this as a high priority for April. We are seeking feedback from the implementation team on the following plan:
## Summary
We think the main user story right now and in the near future that is relevant for Alchemy’s home page is a user coming to Alchemy to visit a specific DAO. Right now, they must sift through the feed and the searchable DAO list to find this DAO. Instead, we should just show them links to DAOs they are a member of directly.
## Product Design
Move the feed to its own tab or remove it completely (I am agnostic about this and open to feedback, but happy to make a call if no one has a strong opinion).

Create a new DAO list showing any DAOs the logged in user holds reputation in. Label this “Your DAOs.” This list should go above the current full, searchable list of DAOs (“All DAOs”).

### PR DESCRIPTION
**Pull request by tibetsprague**
Separate section at the top of the DAOs page with DAOs user is a member of or follows first. Then below that is an Other DAOs section with all other DAOs.
Note that this will currently not show more than 100 DAOs that a user is a member of and 100 that they folllow.

Depends on new client daostack/client#443

Fixes: #1581

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,7 +77,7 @@
     "test:unit": "jest"
   },
   "dependencies": {
-    "3box": "1.16.1",
+    "3box": "1.17.1",
```
Comment (orenyodfat) [inline package.json:80]: is this related to this pr ?

Comment (tibetsprague) [reply package.json:80]: this is from the Dao follow/unfollow PR. it will go away when dev is merged in again

**Comment by tibetsprague** [comment]
Also fixes https://github.com/daostack/alchemy/issues/1585

File: src/lib/daoHelpers.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { DAO, IDAOState, Reputation, Token } from "@daostack/client";
+import { getArc } from "arc";
+import BN = require("bn.js");
+
+// TODO: all this should really be in the client library, after new client is done i will move it there if its not already
```
Comment (orenyodfat) [inline src/lib/daoHelpers.ts:5]: please open an issue in the client for that please.

**Comment by orenyodfat** [comment]
decision : 
-move the search box on top of "Your DAOs" .
-leave only one search box

File: src/lib/daoHelpers.ts (hunk 2 above)
Comment (tibetsprague) [reply src/lib/daoHelpers.ts:5]: https://github.com/daostack/client/issues/447

File: src/components/Daos/DaosPage.tsx [hunk 3]
```diff
@@ -154,69 +182,65 @@ class DaosPage extends React.Component<IProps, IState> {
...
     );
   }
 }
 
+const createSubscriptionObservable = (props: IStateProps, data: SubscriptionData = null) => {
+  const arc = getArc();
+  const { currentAccountAddress, followingDAOs } = props;
+
+  // TODO: right now we don't handle a user following or being a member of more than 100 DAOs
+  //       it was too hard to figure out the UI with infinite scrolling in this case we would need a different UI
```
Comment (dkent600) [inline src/components/Daos/DaosPage.tsx:198]: Is infinite scroll more feasible after the change that has only one search box?

Comment (tibetsprague) [reply src/components/Daos/DaosPage.tsx:198]: no

Comment (tibetsprague) [reply src/components/Daos/DaosPage.tsx:198]: cant really do two lists with infinite scroll on both on the same page easily... unless they are in separate columns or tabs

**Comment by tibetsprague** [comment]
@EzraWeller i switched to one search box at the top of the page. it could definitely still use a bit of stylistic polish

**Comment by arsena21** [comment]
I think we will need a link to the FEED... (see below for inspiration). We don't have a link anywhere else from the home page.

image

**Comment by tibetsprague** [comment]
@arsena21 there is a link in the menu at the bottom left. what if we just moved that menu up to the top left so its more visible?

**Comment by arsena21** [comment]
Yeah, I think that could work

**Comment by orenyodfat** [comment]
does the "Look for dao to follow or join" msg should appears after finding something in the search box ?

Screen Shot 2020-04-21 at 11 45 36

**Comment by arsena21** [comment]
Maybe we should not have the "Your DAOs" block (including the msg Oren posted above around "Look to join or follow below" unless there are any DAOs in that top block?

**Comment by orenyodfat** [comment]
> Maybe we should just hide the "Your DAOs" block (including the msg Oren posted above around "Look to join or follow below" unless there are any DAOs in that top block?

Lets have a quick decisions re that , otherwise it will not get into next release.

**Comment by tibetsprague** [comment]
ok I can hide if there is nothing
@EzraWeller are you down with that? And are you down with the idea to move the sidebar links that are normally at the bottom left to the top left so they are more visible?

**Comment by tibetsprague** [comment]
ok I can hide if there is nothing
@EzraWeller are you down with that? And are you down with the idea to move the sidebar links that are normally at the bottom left to the top left so they are more visible?

**Comment by tibetsprague** [comment]
ok i just went ahead and hid the Your DAOs section if it empty

**Comment by tibetsprague** [comment]
but i didnt move the sidebar links yet
